src/scss/web-components/_web-subscribe.scss

```diff
@@ -25,5 +25,48 @@ web-subscribe {
 
   .g-recaptcha {
     height: 78px;
+
+    &-- {
+      &placeholder {
+        width: 100%;
```

Comment:
Maybe my knowledge is outdated but that feels like a wild interpretation of BEM for me 😄 I know, the CSS is a mess already but I would probably go for `.g-recaptcha__placeholder` and `.g-recaptcha__loader-container` and `.g-recaptcha__loader` and probably drop the `g-re` bit and just use `captcha` as the parent name.